Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -2557,7 +2557,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				INFOPLIST_FILE = XCUITest/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.4;
```

Comment:
@isabelrios Are you sure this is correct? This would bump the minimum iOS requirement from 11.4 to 12.0. I don't think that is unreasonable, but we should probably run that past product and make an informed decision about this with some supporting data. (Last time I checked, we do have _very_ few users with iOS older than 12.0)

Reply:
Same for all the other targets.

Reply:
mmm I think it was updated automatically when I selected a newer swift version to get the snapshot test target to compile.. I will try to set that back to 11.4